packages/web/src/components/buy-sell-modal/SwapTab.tsx

```diff
@@ -79,7 +66,8 @@ export const SwapTab = ({
   availableInputTokens,
   availableOutputTokens,
   onInputTokenChange,
-  onOutputTokenChange
+  onOutputTokenChange,
+  showExchangeRate = false
```

Comment:
this is chill, but do we want a "type" prop so this thing knows if it's a buy/sell/swap? itll make that comment below "show for covert" to make a bit more sense?